app/ui/src/main/java/com/fsck/k9/ui/K9Drawer.kt

```diff
@@ -225,6 +232,8 @@ class K9Drawer(private val parent: MessageList, savedInstanceState: Bundle?) : K
                     .withIcon(folderIconProvider.getFolderIcon(folder.type))
                     .withIdentifier(drawerId)
                     .withTag(folder)
+                    .withSelectedColor(accentColor.and(0xffffff).or(0x11000000))
```

Comment:
This is also something that should be moved to a separate method. Ideally there would be a single method that computes `accentColor` and `selectedColor` and returns a data class with the two values.

You could then use it like this:
```kotlin
val (accentColor, selectedColor) = getDrawerColorsForAccount(account)
```